packages/cli/src/rpc/modules/portal.ts

```diff
@@ -551,52 +559,80 @@ export class portal {
     return true
   }
   // portal_*Ping
-  async historyPing(params: [string]) {
-    const [enr] = params
+  async historyPing(params: [string, number, object | undefined]) {
+    const [enr, ext, payload] = params
     const encodedENR = ENR.decodeTxt(enr)
-    this.logger(`PING request received on HistoryNetwork for ${shortId(encodedENR.nodeId)}`)
-    const pong = await this._history.sendPing(encodedENR, 0)
+    const extension = ext ?? 0
+    this.logger(
+      `PING request received on HistoryNetwork for ${shortId(encodedENR.nodeId)} with extension ${extension}`,
+    )
+
+    let encodedPayload = undefined
+    if (payload !== undefined) {
+      if (typeof payload === 'string') {
```

Comment:
is this necessary?  it shouldn't be a possibility i don't think.  and this line is not included in the other network PING methods

Reply:
No, it shouldn't be.  I was trying to get the RPC tests to work.  Will remove.